pkg/writer/options/options.go

```diff
@@ -5,8 +5,8 @@ import (
 )
 
 type Options struct {
-	Format formats.Format
-	Indent int
+	Format formats.Format `yaml:"format,omitempty" json:"format,omitempty"`
+	Indent int            `yaml:"indent,omitempty" json:"indent,omitempty"`
```

Comment:
I'm curious here, why the yaml annotations here? Any plans? :)

Reply:
i find yaml to be easier to use for a configuration file (comments, readability, templeting and so on..), i skipped this part in this PR because i was not sure we need a configuration file for this CLI.
I can remove it for now.

Reply:
Ah got it, yes I like the idea of a yaml config file too :+1: